x/gamm/pool-models/balancer/pool_test.go

```diff
@@ -1218,3 +1265,91 @@ func TestIsActive(t *testing.T) {
 		})
 	}
 }
+
+func TestCalcJoinPoolNoSwapShares(t *testing.T) {
+	balancerPoolAsset := []balancer.PoolAsset{
+		{Token: sdk.NewInt64Coin("foo", 100), Weight: sdk.NewIntFromUint64(5)},
+		{Token: sdk.NewInt64Coin("bar", 100), Weight: sdk.NewIntFromUint64(5)},
+	}
+
+	tests := map[string]struct {
+		tokensIn        sdk.Coins
+		expNumShare     sdk.Int
+		expTokensJoined sdk.Coins
+		expPoolAssets   sdk.Coins
+		expRemCoin      sdk.Coins
```

Comment:
maybe switch to `expRemainingCoins`, at first I thought this meant "expected removed coins" until I read the test. Just would help with immediate clarity :)

Reply:
ah yeah this is a good point but `RemCoins` is pulled from the convention we use in the function itself so we'd have to make the change there. might be worth making a small issue/pr for readability there

Reply:
changed anyway for test readability!